### PR TITLE
Update doc test README

### PR DESCRIPTION
**Pull request by frankbu**

### CODE REVIEW

File: tests/README.md [hunk 1]
```diff
@@ -1,89 +1,140 @@
 # Testing istio.io Content
 
 This folder contains tests for the content on [istio.io](http://istio.io).
+More specifically, these test confirm that example and task documents, which contain
```
Comment (ericvn) [inline tests/README.md:4]:
```suggestion
More specifically, these tests confirm that the example and task documents, which contain
```

File: tests/README.md [hunk 2]
```diff
@@ -1,89 +1,140 @@
...
+    ```golang
+    func TestMain(m *testing.M) {
+    framework.NewSuite("my-istioio-test", m).
+        SetupOnEnv(environment.Kube, istio.Setup(&ist, nil)).
+        RequireEnvironment(environment.Kube).
+        Run()
+    }
+    ```
 
 1. To create a test, you use `istioio.Builder` to build a series of steps that will
```
Comment (ericvn) [inline tests/README.md:67]:
```suggestion
1. To create a test, you use `istioio.NewBuilder` to build a series of steps that will
```

**Comment by frankbu** [comment]
Failed doc test has nothing to do with this PR, so I think it's ready to merge.

**Pull request merged**